randovania/games/am2r/exporter/patch_data_factory.py

```diff
@@ -38,6 +38,10 @@ def _create_pickups_dict(self, pickup_list: list[ExportedPickupDetails], item_in
         for pickup in pickup_list:
             quantity = pickup.conditional_resources[0].resources[0][1]
             object_name = self.game.region_list.node_from_pickup_index(pickup.index).extra["object_name"]
+            text_index = 1 if (pickup.original_pickup.resource_lock and
```

Comment:
Please use `is not None`. I prefer to be more explicit when checking for the singleton.
Writing it that way can be a gotcha as things like containers can evaluate to false if there length is zero.

Reply:
Pycharm at one pointed started yelling at me for writing stuff as `is not none` instead of like this, so through this'd be better. good to know.